### DISCUSSION TITLE
Odd rendering of cross-references in Safari (blue boundaries)

### DISCUSSION THREAD

**Discussion by wjakob** [pradyunsg/furo#740] [Potential Issue] [answered]
I've recently been noticing an odd HTML rendering issue when viewing Sphinx/Furo documentation on the Safari browser on macOS.  (This is with furo 2023.9.10 and Sphinx 7.2.6.). My browser is Safari 17.1 on Ventura 13.6.2.

Here is a picture of what the documentation of a particular function looks like on Chrome. This is also how it used to look like on Safari.
<img width="778" alt="Screenshot 2023-12-15 at 06 50 48" src="https://github.com/pradyunsg/furo/assets/1203629/fd53cc72-487a-47ff-821e-6de50d76dd83">

And this is what it looks like now, on Safari. Note that there is a bright blue border around on one of the cross-references. In some parts of my project's documentation, entire pages are full of these bright blue bubbles, and it quite distracting.
<img width="759" alt="Screenshot 2023-12-15 at 06 55 23" src="https://github.com/pradyunsg/furo/assets/1203629/b4cadf09-1f84-42d2-9b82-598a881a498d">

Here is a link to the project page on readthedocs, which shows this issue as well: https://drjit.readthedocs.io/en/nanobind_v2/reference.html. Curiously, the set of "bubbles" there is actually different from those in my offline build 🤔

**Comment by pradyunsg** [comment]
Huh, this looks like a Safari bug. Weirdly, it becomes correct if I zoom into the page but the inspector conveys that the color should be the light grey that the stylesheet say.

<img width="1060" alt="Screenshot 2023-12-20 at 16 28 09" src="https://github.com/pradyunsg/furo/assets/3275593/9ca82bcc-6cc1-42f2-998a-21e3acbd5ed6">

Except, that the computed value matches the text color until I zoom in when it corrects itself?

**Reply by wjakob** [reply]
Agreed that it seems like a Safari bug. Though zooming in/out does not fix the issue on my end.

Given that Safari is a relatively widely used browser, would it make sense to work around the issue somehow? (I am not a HTML/CSS expert, so I don't know how feasible that is)

**Reply by wjakob** [reply]
Is there anything that I can do to help get this fixed? Having these bubbles randomly appear makes for a poor reading experience when using Safari.

**Reply by wjakob** [reply]
@pradyunsg  -- I wanted to follow up on this issue. This post is marked as "Needs information to reproduce/better understand the issue". Is that the case? (At least for the reproduction part, it seemed to me that you could also observe the issue when using Safari.)

**Reply by pradyunsg** [reply]
Heyo! Yea, this need information/investigation into why this happens on Safari and how to diagnose this issue.

**Reply by wjakob** [reply]
My knowledge of HTML/CSS is very rudimentary, so I don't think I can be of much help sadly :-/. Is this something you could help with?

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/issues/819 filed for this. It might be a Safari bug but that'll need investigation.
